obal/data/roles/diff_brew_package/tasks/main.yml

```diff
@@ -1,121 +1,15 @@
 ---
-- name: get spec file
-  find:
-    pattern: "*.spec"
-    path: "{{ inventory_dir }}/packages/{{ inventory_hostname }}"
-  register: spec_file
+- name: 'Include diff-brew-package'
+  include_tasks: diff.yml
+  when: not diff_brew_package_skip
   tags:
-    - always
+    always
```

Comment:
can you make this `- always` again? technically both work, but we use the explicit dashed form everywhere else